wallet/build.gradle

```diff
@@ -112,6 +129,7 @@ android {
         exclude 'org.bitcoin.test.checkpoints.txt'
         exclude 'messages.properties'
         exclude 'META-INF/services/sun.net.spi.nameservice.NameServiceDescriptor'
+        exclude 'META-INF/services/org.xmlpull.v1.XmlPullParserFactory'
```

Comment:
Does putting this in solve the issues when building?  I used to get about 39 or 40 errors from `xmlpull`

Reply:
Yes, it stop warning about `xmlpull`.

Reply:
Actually, as you just mentioned below, it doesn't stop to warn about `xmlpull`, it prevents a compiling error.